lib/oga/xml/node.rb

```diff
@@ -55,6 +55,21 @@ def children=(nodes)
         end
       end
 
+      # Adds a child (or creates a child text node) to this node.
+      #
+      # @param [Oga::XML::Node|String] child
+
+      def <<(child)
+        child = Text.new(:text => child) if child.is_a?(String)
+
+        unless child.is_a?(Node)
+          raise ArgumentError, 'Can only add nodes or text.'
+        end
+
+        children << child
+        self
+      end
+
```

Comment:
> For appending nodes to a parent, I'll note that all of REXML Nokogiri and libxml-ruby do support appending nodes to parents via the << operator.

As mentioned in another comment, implementing something in Oga just because other libraries do it is not a good reason. I want convincing use cases and I want to be certain additions are useful to all users.

> In practice, I don't think it's ambiguous what the operator does. Reading code from the other three libs doesn't cause any cognitive friction for me in this particular case.

This is subjective, and my opinion differs. I prefer the more explicit `some_node.children << foo` and I'm not convinced adding support for `some_node << foo` is worth the extra code, tests, maintenance burden, etc.